docs/administration/project_organization/bundle_extension/package_structure.md

```diff
@@ -4,51 +4,44 @@ description: All code contributions to Ibexa DXP must follow package and bundle
 
 # Package and bundle structure and namespaces
 
-If you wish to contribute to [[= product_name =]] development,
-you need to adhere to the package and bundle structure and namespace standards.
+Bundle allows you to extend the system by adding new services, routes, controllers, and views, and extend functionality to fully customize your [[= product_name =]] application.
+If you want to contribute to [[= product_name =]] development, you need to adhere to the package and bundle structure and namespace standards.
 
-The following conventions apply to contributions to [[= product_name_base =]] core code, not to third party packages.
+All conventions described here apply to contributions to [[= product_name_base =]] core code, not to third party packages.
 
 !!! note
-
     New code needs to follow the rules outlined here.
     They are being applied progressively to existing code.
-
 ## Root PHP namespace
 
 Define [[= product_name =]] core PHP code in a namespace with the following prefix:
-
 ```php
 namespace Ibexa;
 ```
 
 A package which groups some DXP features can use an additional prefix, for example:
-
 ```php
 namespace Ibexa\Commerce;
-```
-
 ```php
 namespace Ibexa\Personalization;
 ```
 
 ## Packages
 
-The general package directory structure and corresponding PHP namespace mapping are:
-
+The general package directory structure and corresponding PHP namespace mappings are:
```

Comment:
```suggestion
The general package directory structure and corresponding PHP namespace mappings are as follows:
```
This structure uses different formatting than the one in the create bundle article. Could we decide on one and use it consistently?